master/buildbot/test/fake/fakedb.py

```diff
@@ -90,6 +91,10 @@ def __init__(self, **kwargs):
         for k, v in iteritems(self.values):
             if isinstance(v, str):
                 self.values[k] = unicode(v)
+        # encode columns that should be stored utf-8 encoded (e.g. LargeBinary
+        # columns) to utf-8
+        for col in self.utf8_columns:
+            self.values[col] = self.values[col].encode("utf-8")
```

Comment:
No, this is emulation of Binary columns --- they are read and written as bytestrings.